roles/tas_single_node/defaults/main.yml

```diff
@@ -19,7 +19,14 @@ tas_single_node_skip_os_install: false
 
 tas_single_node_rekor_templates:
   - manifests/rekor/redis-server.yaml
-  - manifests/rekor/rekor-server.yaml
+  - manifests/rekor/rekor-server.j2
+
+tas_single_node_rekor_redis:
+  database_deploy: true
+  redis:
+    address: "rekor-redis"
+    port: 6379
+    password: ""
```

Comment:
Let's provide a default password for Ansible-managed redis while we're at it, just to make sure we have something (and be consistent with the MariaDB defaults)